src/sdk/model/sso/saml-identity-provider-config-request.ts

```diff
@@ -0,0 +1,200 @@
+import { SAMLIdentityProviderConfigRequestJson } from './__json__/saml-identity-provider-config-request-json';
+import { SAMLNameIdPolicyFormat } from './__json__/saml-name-id-policy-format';
+import { SAMLPrincipalType } from './__json__/saml-principal-type';
+import { SAMLSignatureAlgorithm } from './__json__/saml-signature-algorithm';
+import { SAMLSignatureKeyName } from './__json__/saml-signature-key-name';
+
+/* istanbul ignore next: autogenerated */
+export class SAMLIdentityProviderConfigRequest {
+
+  private readonly _json: SAMLIdentityProviderConfigRequestJson;
+
+  constructor(immutableSAMLIdentityProviderConfigRequest: SAMLIdentityProviderConfigRequest);
+  constructor(immutableSAMLIdentityProviderConfigRequestJson: SAMLIdentityProviderConfigRequestJson);
+  constructor(firstParam: SAMLIdentityProviderConfigRequest | SAMLIdentityProviderConfigRequestJson) {
+    // Parameters constructor
+    if (firstParam instanceof SAMLIdentityProviderConfigRequest) {
+      // Copy constructor
+      this._json = (firstParam as SAMLIdentityProviderConfigRequest).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as SAMLIdentityProviderConfigRequestJson;
+    }
+  }
+
+  /**
+   * Get enforce sso.
+   * @returns {boolean}
+   */
+  get enforceSso(): boolean {
+    return this._json.enforce_sso;
+  }
+
+  /**
+   * Get enabled.
+   * @returns {boolean}
+   */
+  get enabled(): boolean {
+    return this._json.enabled;
+  }
+
+  /**
+   * Get single sign on service url.
+   * @returns {string}
+   */
+  get singleSignOnServiceUrl(): string {
+    return this._json.single_sign_on_service_url;
+  }
+
+  /**
+   * Get single logout service URL.
+   * @returns {string | undefined}
+   */
+  get singleLogoutServiceUrl(): string | undefined {
+    return this._json.single_logout_service_url;
+  }
+
+  /**
+   * Get name id policy format.
+   * @returns {SAMLNameIdPolicyFormat}
+   */
+  get nameIsPolicyFormat(): SAMLNameIdPolicyFormat | undefined {
+    return this._json.name_id_policy_format;
+  }
+
+  /**
+   * Get principal type.
+   * @returns {SAMLPrincipalType}

```

Comment:
| undefined  return doc for all getters in this class that have `undefined` possible return type

Reply:
Fixed